packages/frontend/component/src/ui/notification/mobile/notification-card.tsx

```diff
@@ -16,7 +17,14 @@ export function MobileNotificationCard({
     icon = <InformationFillDuotoneIcon />,
     iconColor,
     onDismiss,
+    error,
   } = notification;
+  const t = useI18n();
+  const errorI18nKey = error ? (`error.${error.name}` as const) : undefined;
```

Comment:
will this have impact on unused i18n key strings detection?

Reply:
i think all i18n keys starting with `error.` come from codegen?